modules/ROOT/pages/monitoring/logging.adoc

```diff
@@ -1308,6 +1308,10 @@ For `event=start`, the value is `00000000`.
 | executableQueryCacheHit
 | Whether the query string matched a cached execution plan.
 
+| queryLang
+| Cypher version.
+Possible values are `CYPHER 5` and `CYPHER 25`.
```

Comment:
Or should we just say 'Cypher versions' for now?

Reply:
Sorry for the delay, personally I prefer the term "Query language version". We might get a "GQL strict" language at some point.

Reply:
...but if you feel "Cypher version" is clearer, go with that!